bundles/admin/admin-announcements/view/AnnouncementsList.jsx

```diff
@@ -18,21 +18,21 @@ const StyledButton = styled(Button)`
 `;
 const AnnouncementsList = ({controller,  announcements, title, activeKey }) => {
 
+  const lang = Oskari.getLang();
   const callback = (key) => {
     controller.openCollapse(key);
   }
   return (
     <div>
         <div>
         <Collapse accordion activeKey={activeKey} onChange={callback}>
-          {announcements.map((form, index) => {
+          {announcements.map((announcement, index) => {
             return (
-              <Panel header={form.title ? form.title : Oskari.getMsg('admin-announcements', 'addNewForm')} key={index} >
+              <Panel header={announcement.locale[lang] && announcement.locale[lang].name && announcement.locale[lang].name ? announcement.locale[lang].name : Oskari.getMsg('admin-announcements', 'addNewForm')} key={index} >
```

Comment:
Spread the code out a bit so it's easier to read than trying to cram everything in one line and inside JSX :)
```suggestion
    {announcements.map((announcement, index) => {
          const loc = Oskari.getLocalized(announcement.locale) || {};
          const title = loc.name ||  Oskari.getMsg('admin-announcements', 'addNewForm');
          return (
              <Panel header={title} key={index} >
```